src/main/webapp/new-post.html

```diff
@@ -99,7 +99,7 @@ <h1 class="site-heading text-center text-white d-none d-lg-block">
           </div>
           <div id="map"></div>
           <div id="infowindow-content">
-            <img src="" width="16" height="16" id="place-icon">
+            <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=" width="16" height="16" id="place-icon">
```

Comment:
Actually I'm not very sure where/when this happens or how it look when it displays a transparent image?

Reply:
Ah sorry I should have explained a little more cause I admit this is a littler weird and hacky. The problem is this image is used in the location marker to show an icon for the place. Originally with no `src` defined, it was showing a small missing picture icon before selecting a location, which didn't look very good.

This just replaces the empty `src` with a 1x1 transparent picture generated from here: http://png-pixel.com/. It was the simplest way to hide the problem without digging too deep into how the code uses this picture.